docs/concepts/automations.md

```diff
@@ -132,6 +90,205 @@ Prefect tries to infer the relevant event whenever possible, but sometimes one d
 
 Specify a name and, optionally, a description for the automation.
 
+## Custom triggers
+
+Custom triggers allow advanced configuration of the conditions on which a trigger executes its actions. Several custom trigger fields accept values that end with trailing wildcards, like `"prefect.flow-run.*"`.
+
+![Viewing a custom trigger for automations for a workspace in Prefect Cloud.](/img/ui/automations-custom.png)
+
+The schema that defines a trigger is as follows:
+
+| Name               | Type               | Supports trailing wildcards | Description |
+| ------------------ | ------------------ | --------------------------- | ----------- |
+| **match**          | object             | :material-check:            | Labels for resources which this Automation will match. |
+| **match_related**  | object             | :material-check:            | Labels for related resources which this Automation will match. |
+| **after**          | array of strings   | :material-check:            | Event(s), one of which must have first been seen to start this automation. |
+| **expect**         | array of strings   | :material-check:            | The event(s) this automation is expecting to see. If empty, this automation will evaluate any matched event. |
+| **for_each**       | array of strings   | :material-close:            | Evaluate the Automation separately for each distinct value of these labels on the resource. By default, labels refer to the primary resource of the triggering event. You may also refer to labels from related resources by specifying `related:<role>:<label>`. This will use the value of that label for the first related resource in that role. |
```

Comment:
```suggestion
| **for_each**       | set of strings   | :material-close:            | Evaluate the Automation separately for each distinct value of these labels on the resource. By default, labels refer to the primary resource of the triggering event. You may also refer to labels from related resources by specifying `related:<role>:<label>`. This will use the value of that label for the first related resource in that role. |
```